contracts/truefi2/SpotBaseRateOracle.sol

```diff
@@ -36,4 +44,10 @@ contract SpotBaseRateOracle {
         (, , , , uint128 currentVariableBorrowRate, , , , , , , ) = aaveLendingPool.getReserveData(asset);
         return uint256(currentVariableBorrowRate).div(1e23);
     }
+
+    function _getCompoundBorrowAPY(address asset) internal view returns (uint256 apy) {
```

Comment:
We'd rather avoid inplicit returns, just use `return` in the end of the function :)